docs/source/dcos-docker-cli.rst

```diff
@@ -369,3 +369,14 @@ CLI Reference
 
 .. click:: cli.dcos_docker:download_artifact
   :prog: dcos-docker download-artifact
+
```

Comment:
❓ What do you think of adding a section in this document on using loopback sidecars?

Reply:
For example noting that one sidecar is used by all containers.
This even confused @jieyu I think.

Reply:
Adding additional documentation.